src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random randomGenerator = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+
+    public Figure getRandomFigure() {

```

Comment:
Consider renaming the method `getColorSupplier()` in the `ColorSupplier` class to `getRandomColor()` to better reflect its purpose, as per the task description.